main.nf

```diff
@@ -14,14 +14,15 @@ params.group_col = 'Group_Number'
 params.folder_col = 'PathName_Orig'
 params.file_col = 'FileName_Orig'
 params.shard_col = "Shard_Id"
+params.experiment_file = 'Experiment.txt'
```

Comment:
With the default params included in `nextflow.config`, they can be removed here.